src/diagnostics/riscv64/disasm-riscv64.cc

```diff
@@ -1577,6 +1583,10 @@ void Decoder::DecodeIType(Instruction* instr) {
 }
 
 void Decoder::DecodeSType(Instruction* instr) {
+      
+  if (switch_sew(instr) != -1){
+     DecodeRvvVS(instr);
+  }
   switch (instr->InstructionBits() & kSTypeMask) {
```

Comment:
add a else.
If not it wiil run DecodeRvvVS(instr) and ` switch (instr->InstructionBits() & kSTypeMask) {`